package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@edx/edx-proctoring",
   "//": "Be sure to update the version number in edx_proctoring/__init__.py",
   "//": "Note that the version format is slightly different than that of the Python version when using prereleases.",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
we want to keep this in lock-step with the version in the __init__ file; it was behind because someone outside our team merged without reading our comments to that effect :)